SERVICE_WORKER_CODE.php

```
@@ -736,136 +715,43 @@ function ps_enhance_images( $context ) {
-            ? implode( ', ', $product_data['categories'] ) 
-            : '';
-        $enhancement = isset( $context['enhancement'] ) ? $context['enhancement'] : 'default';
-        
-        // Build image generation prompt based on enhancement style
-        $prompt = "Professional product photography of {$product_name}";
-        
-        if ( $categories ) {
-            $prompt .= " ({$categories})";
-        }
+        $image_url = isset( $product_data['image_url'] ) ? $product_data['image_url'] : '';
+        $enhancement = isset( $context['enhancement'] ) ? $context['enhancement'] : 'remove_bg';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$enhancement'. (undefined)

(UnusedLocalVariable)